lib/db/schema/patch-073-074.sql

```diff
@@ -0,0 +1,179 @@
+SET NAMES utf8mb4 COLLATE utf8mb4_bin;
+
+CREATE TABLE IF NOT EXISTS devicesCapabilities (
+  uid BINARY(16) NOT NULL,
+  deviceId BINARY(16) NOT NULL,
+  capability TINYINT UNSIGNED NOT NULL,
+  PRIMARY KEY (uid, deviceId, capability),
+  FOREIGN KEY (uid, deviceId) REFERENCES devices(uid, id) ON DELETE CASCADE
```

Comment:
*Sigh*, I should have used `ON DELETE CASCADE` on the `devices` table itself. And `unverifiedTokens`. :man_facepalming: